packages/transformer/src/test/standalone/IModelTransformerHub.test.ts

```diff
@@ -3593,8 +3586,12 @@ describe("IModelTransformerHub", () => {
     ) => {
       transformer["_options"]["branchRelationshipDataBehavior"] =
         "unsafe-migrate";
-      transformer["_options"]["unsafeFallbackReverseSyncVersion"] = ";2";
-      transformer["_options"]["unsafeFallbackSyncVersion"] = ";3";
+      transformer["_options"][
+        "argsForProcessChanges"
+      ]!.unsafeFallbackReverseSyncVersion = ";2";
```

Comment:
Can you use ["unsafeFallBackReverseSyncVersion"] instead? It gives us type safety on private vars

Reply:
Sounds good